docs/sources/flow/reference/components/prometheus.integration.process_exporter.md

```diff
@@ -0,0 +1,108 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.​integration.​process_exporter
+---
+
+# prometheus.integration.process_exporter
+The `prometheus.integration.process_exporter` component embeds
+[process_exporter](https://github.com/ncabatoff/process-exporter) for collecting process stats mined from `/proc`.
+
+## Usage
+
+```river
+prometheus.integration.process_exporter "LABEL" {
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`procfs_path`       | `string`                 | procfs mountpoint. | `/proc` | no
+`track_children`    | `bool`                   | If a proc is tracked, also track any of it's children that are not part of their own group. | `true` | no
+`track_threads`     | `bool`                   | Report on per-threadname metrics as well. | `true` | no
+`gather_smaps`      | `bool`                   | Gather metrics from the smaps file, which contains proportional resident memory size. | `true` | no
+`recheck_on_scrape` | `bool`                   | Recheck process names on each scrape. | `true` | no
+
+## Blocks
+The following blocks are supported inside the definition of `prometheus.integration.process_exporter`:
+
+Hierarchy        | Block      | Description | Required
+---------------- | ---------- | ----------- | --------
+process_names          | [process_names][]  | A collection of matching rules to use for deciding which processes to monitor. Each config can match multiple processes, which will be tracked as a single process "group." | no
```

Comment:
This description is also pretty long, you can make this shorter like `Rules for deciding which processes to monitor.` and then move the longer description to the intro sentence of the `process_names block` section.